docs/theme-context.rst

```diff
@@ -0,0 +1,123 @@
+Read the Docs data passed to Sphinx build context
+=================================================
+
+Before calling `sphinx-build` to render your docs, Read the Docs injects some
+extra context in the templates by using the `html_context Sphinx setting`_ in
+the ``conf.py`` file. This extra context allows Read the Docs to add some
+features like "Edit on GitHub" and use a user custom Analytics code, among others.
```

Comment:
Instead of describing the specific features, it's probably easiest to just say something along the lines of:

```This extra context is used by the Read the Docs Sphinx theme to add additional features to the built documentation.```